frontend/src/utils/profile.tsx

```diff
@@ -216,3 +216,18 @@ export function toDateString(date: Date): string {
   const month = date.getUTCMonth() + 1
   return `${date.getUTCFullYear()}-${month}-${date.getUTCDate()}`
 }
+
+export function parseBirthday(birth_date: Nullable<Date>): Nullable<Date> {
+  console.log("birth_date:", birth_date)
+  let birthday = null
+  if (birth_date !== null) {
+    birthday = new Date(birth_date)
+    // We have Django configured to store dates using local time.
+    // Date objects assume the date is UTC when parsing so we add 1
+    // to correct for the EST/EDT - UTC timezone difference
+    birthday.setDate(birthday.getDate() + 1)
+  }
+  console.log("birthday:", birthday)
+
+  return birthday
+}
```

Comment:
So I'm not sure what happened but this function should have been deleted. The dates on the backend aren't really aware of timezone since we only store the date and not the time. The `toDateString()` function above interprets the date as UTC and avoids having to add one to adjust for the timezone and should be used in place of `Date.toLocaleDateString()` for displaying the birth date. This function should be removed.

Reply:
sounds good